app/components/QuoteForm.tsx

```diff
@@ -0,0 +1,104 @@
+// LIBRARY IMPORTS
+import {
+  VStack,
+  FormControl,
+  Input,
+  Textarea,
+  Button,
+  Text,
+} from "@chakra-ui/react";
+
+// TYPE DEFINTIONS
+interface QuoteFormProps {
+  formTitle: string;
+  field1Placeholder: string;
+  field2Placeholder: string;
+  field3Placeholder: string;
+  submitButtonText: string;
+}
+
+export default function QuoteForm({
+  formTitle,
+  field1Placeholder,
+  field2Placeholder,
+  field3Placeholder,
+  submitButtonText,
+}: QuoteFormProps) {
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    // Form submission logic goes here
+  };
+
+  return (
+    <form onSubmit={handleSubmit}>
+      <Text textAlign="center" fontSize="3xl" mt={8}>
+        {formTitle}
+      </Text>
+      <VStack
+        spacing={6}
+        px={8}
+        py={10}
+        bg="white"
+        borderRadius="lg"
+        boxShadow="xl"
+      >
+        <FormControl isRequired>
+          <Input
+            borderColor="brand.backgroundDark"
+            borderRadius="full"
```

Comment:
I made some updates to this but not matching the wireframes - mostly just so there are color changes on select and on hover for accessibility but I matched the colors with the brand colors. In terms of the grey placeholder text, I think the shade of grey Kayla added to the wireframe is arbitrary as its not part of the design system so I think we can just stick with the default grey text?

Reply:
There is a grey color in the design system called Accent (HEX 787373), for some reason it's not included in theme colors. There's also green Accent (HEX 32CD32) that's also not in there. 

Thanks for changing border color for select and on hover. I'm not sure how I feel about red border on hover; in general, I think it looks good and I agree that there should be change of color for all the different states, but I think that it would be best to ask Kayla what different states should look like. I'm okay if it doesn't make into this PR, but something that needs to be addressed at some point.

Reply:
Good call, I think it's because we made the brand theme colors before Kayla had created the explicit Design System. I updated it to use the grey and green and removed the red, which you're right looks off as it usually signifies an error. We can ask Kayla for final colors